add a fast path for EmbeddingBag calling FBGEMM

xing-liu opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#36679 add a fast path for EmbeddingBag calling FBGEMM**

Differential Revision: [D21058034](https://our.internmc.facebook.com/intern/diff/D21058034)

Commit:
add a fast path for EmbeddingBag calling FBGEMM

[ghstack-poisoned]

Commit:
Update on "add a fast path for EmbeddingBag calling FBGEMM"




[ghstack-poisoned]

jianyuh commented on aten/src/ATen/native/EmbeddingBag.cpp:
```diff
@@ -5,7 +5,11 @@
 
 #include <TH/THBlasUtils.h>
 
+#if defined(USE_FBGEMM)
```
Nit: for simplicity:
```
#ifdef USE_FBGEMM
```

xing-liu committed:
Update on "add a fast path for EmbeddingBag calling FBGEMM"


Differential Revision: [D21058034](https://our.internmc.facebook.com/intern/diff/D21058034)

[ghstack-poisoned]

xing-liu committed:
Update on "add a fast path for EmbeddingBag calling FBGEMM"


Differential Revision: [D21058034](https://our.internmc.facebook.com/intern/diff/D21058034)

[ghstack-poisoned]

jianyuh commented:
@yinghai , @supriyar Could you have a check if this looks good to you? Thanks!

supriyar reviewed (approved):
LGTM, thanks!
